readthedocs/oauth/utils.py

```diff
@@ -31,9 +31,17 @@ def update_webhook(project, integration, request=None):
 
     updated = False
     try:
-        account = project.remote_repository.account
-        service = service_cls(request.user, account)
-        updated, __ = service.update_webhook(project, integration)
+        remote_relations = project.remote_repository.remote_relations.filter(
+            account__isnull=False
+        )
+
+        for relation in remote_relations:
+            service = service_cls(request.user, relation.account)
+            updated, __ = service.update_webhook(project, integration)
+
+            if updated:
+                break
```

Comment:
I'm not entirely sure if we want to loop through all the remote relations and try to use the accounts for each user to activate the webhook, I think this will make sure webhooks are updated even if the user who imported the repo doesn't have a valid account. would like to know your thoughts?

Reply:
Yeah. I think it's weird that if the user that imports the project doesn't have access to setup the webhook, it's done using a different user. That sounds bad, IMO.

I suppose we need to use the current logged in user for this action, and if it fails, that user should contact an admin of that repository to "re-sync" the webhook. I'd say that we should filter the previous query by using `user=request.user`.